Automattic/wp-calypso
# Woo: Add variations to product create action-list

This adds variations to the action-list for product create.

To Test:

1. Run `npm test`
2. Run `npm start`
3. Visit `http://calypso.localhost:3000/store/product/<site url>`
4. Fill out name
5. Add old and new categories
5. Add a couple of variations
6. Click Save
7. Visit wp-admin to verify that the category and product have been created correctly.
